hack/multus/multus.yaml

```diff
@@ -57,7 +57,7 @@ metadata:
   name: multus
   namespace: kube-system
 ---
-apiVersion: extensions/v1beta1
```

Comment:
this should not work. with apps/v1, you have to specify `selector`: https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/#create-a-daemonset